backend/src/routes/api/notebooks/index.ts

```diff
@@ -70,7 +70,7 @@ module.exports = async (fastify: KubeFastifyInstance) => {
     ),
   );
 
-  fastify.patch(
```

Comment:
I think we want to keep patch -- I missed that you turned delete into a full put.